packages/crashlytics/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -56,9 +59,67 @@ public void deleteUnsentReports() {
         getFirebaseCrashlyticsInstance().deleteUnsentReports();
     }
 
-    public void recordException(String message, JSArray stacktrace) {
+    public void recordException(String message, JSArray stacktrace, JSArray customProperties) {
```

Comment:
```suggestion
    public void recordException(String message, JSArray stacktrace, @Nullable JSArray customProperties) {
```

Let's add the `@Nullable`  annotation to make it easier to maintain.